akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -37,16 +37,29 @@ object ExceptionHandler {
         if (!knownToBeSealed) ExceptionHandler(knownToBeSealed = true)(this orElse default(settings)) else this
     }
 
+  private def log(verbosity: ErrorLoggingVerbosity, simple: ⇒ Unit, full: ⇒ Unit) =
+    verbosity match {
+      case ParserSettings.ErrorLoggingVerbosity.Off    ⇒
+      case ParserSettings.ErrorLoggingVerbosity.Simple ⇒ simple
+      case ParserSettings.ErrorLoggingVerbosity.Full   ⇒ full
+    }
+
   def default(settings: RoutingSettings): ExceptionHandler =
     apply(knownToBeSealed = true) {
       case IllegalRequestException(info, status) ⇒ ctx ⇒ {
-        ctx.log.warning(
-          "Illegal request {}\n\t{}\n\tCompleting with '{}' response",
-          ctx.request, info.formatPretty, status)
+        log(
+          settings.errorLoggingVerbosity,
+          ctx.log.warning("Illegal request completing with '{}' response", status),
```

Comment:
Because this warning will be used for Simple which is meant to be one liner. At least it is how this is used in other places of akka-http.